### PR TITLE
small change to chunk part numbering

### PR DESCRIPTION
**Pull request by boris-dimitrov**
since the number of parts can be determined in advance, we can use a slightly
more conventional naming scheme and still have lexicographic order

also adding some rudimentary validation

### CODE REVIEW

File: idseq_pipeline/commands/non_host_alignment_functions.py [hunk 1]
```diff
@@ -501,15 +501,26 @@ def check_s3_file_presence(s3_path):
       return 0
 
 # job functions
-def chunk_input(input_files_basenames, chunk_nlines, part_suffix):
+def chunk_input(input_files_basenames, chunk_nlines, chunksize):
```
Comment (boris-dimitrov) [inline idseq_pipeline/commands/non_host_alignment_functions.py]: Btw I tested this function in a standalone python interpreter.   Caught one bug, fixed.

File: idseq_pipeline/commands/non_host_alignment_functions.py [hunk 2]
```diff
@@ -605,8 +616,7 @@ def run_gsnapl_remotely(input_files, lazy_run):
     remote_index_dir = "%s/share" % remote_home_dir
     # split file:
     chunk_nlines = 2*GSNAPL_CHUNK_SIZE
-    part_suffix = "-chunksize-%d-part-" % GSNAPL_CHUNK_SIZE
```
Comment (cdebourcy) [inline idseq_pipeline/commands/non_host_alignment_functions.py]: This broke it, part_suffix is also used in run_gsnapl_remotely

File: idseq_pipeline/commands/non_host_alignment_functions.py [hunk 3]
```diff
@@ -708,8 +718,7 @@ def run_rapsearch2_remotely(input_fasta, lazy_run):
     remote_index_dir = "%s/references/nr_rapsearch" % remote_home_dir
     # split file:
     chunk_nlines = 2*RAPSEARCH_CHUNK_SIZE
-    part_suffix = "-chunksize-%d-part-" % RAPSEARCH_CHUNK_SIZE
```
Comment (cdebourcy) [inline idseq_pipeline/commands/non_host_alignment_functions.py]: part_suffix is also used in run_rapsearch2_remotely

Comment (boris-dimitrov) [reply idseq_pipeline/commands/non_host_alignment_functions.py]: oh sorry

Comment (cdebourcy) [reply idseq_pipeline/commands/non_host_alignment_functions.py]: No worries :)

**Pull request merged**